client/my-sites/migrate/section-migrate.jsx

```diff
@@ -72,6 +73,32 @@ class SectionMigrate extends Component {
 		this.props.navigateToSelectedSourceSite( sourceSiteId );
 	};
 
+	getStartTime = () => {
+		if ( ! window.localStorage ) {
```

Comment:
If we're going to use `localStorage`, I wonder if we should use [lib/browser-storage](https://github.com/Automattic/wp-calypso/tree/81162c9e9c8c2be8e78cac04e4c8c8735963434a/client/lib/browser-storage) instead.

That being said, I have a feeling that we should probably return this value in the API response. See p1575924542382600-slack-importing

Reply:
I think you're right about the API - I'll see about adding the value there.